tests/system/providers/trino/example_trino_query.sql

```diff
@@ -0,0 +1,20 @@
+/*
+ Licensed to the Apache Software Foundation (ASF) under one
+ or more contributor license agreements.  See the NOTICE file
+ distributed with this work for additional information
+ regarding copyright ownership.  The ASF licenses this file
+ to you under the Apache License, Version 2.0 (the
+ "License"); you may not use this file except in compliance
+ with the License.  You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing,
+ software distributed under the License is distributed on an
+ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ KIND, either express or implied.  See the License for the
+ specific language governing permissions and limitations
+ under the License.
+*/
+
+SELECT * FROM {{ params.SCHEMA }}.{{ params.TABLE }}
```

Comment:
I think this file is redundant. You can just place this query in the sql parameter of the operator being used.
There is no reason to test/show example of how to use templating from `.sql` file in TrinoOperator example.
It's also hard for people who will read the example from the docs as they won't see what .sql file contains.

Reply:
Removed the redundant file